src/lib/assets/icons/index.ts

```diff
@@ -38,6 +38,7 @@ import LogoTurborepo from './logo-turborepo.svg?component';
 import LogoV0 from './logo-v0.svg?component';
 import LogoVercelCircle from './logo-vercel-circle.svg?component';
 import LogoVercel from './logo-vercel.svg?component';
+import MagnifyingGlass from './magnifying-glass.svg';
```

Comment:
We are taking advantage of `@poppanator/sveltekit-svg` to be able to import `.svg` files as Svelte components.

```suggestion
import MagnifyingGlass from './magnifying-glass.svg?component';
```

Reply:
Not sure how I missed this thanks